fabtests/common/shared.c

```diff
@@ -3216,6 +3217,37 @@ void ft_parse_opts_range(char* optarg)
 	test_size = range_test_size;
 }
 
+void ft_parse_opts_list(char* optarg)
+{
+	int i, ret;
+	char *token;
+
+	optarg += 2; // remove 'l:'
+	test_cnt = 1;
+	for (i=0; optarg[i] != '\0'; i++) {
+		test_cnt += optarg[i] == ',';
+	}
+	range_test_size = calloc(test_cnt, sizeof(*range_test_size));
```

Comment:
My guess is we need another variable `list_test_size`

Reply:
it's called `test_cnt`

Reply:
He's referring to range_test_size -> list_test_size.  I think we need to rename range_test_size to user_test_sizes.